win/CS/HandBrake.ApplicationServices/Interop/HbLib/HbFunctions.cs

```diff
@@ -362,6 +362,14 @@ public static class HBFunctions
         [DllImport("hb.dll", EntryPoint = "hb_filter_init", CallingConvention = CallingConvention.Cdecl)]
         public static extern IntPtr hb_filter_init(int filter_id);
 
+        ///hb_filter_param_t* hb_filter_param_get_presets(int filter_id);
+        [DllImport("hb.dll", EntryPoint = "hb_filter_param_get_presets", CallingConvention = CallingConvention.Cdecl)]
+        public static extern IntPtr hb_filter_param_get_presets(int filter_id);

```

Comment:
The JSON version of these calls are already implemented. See hb_filter_get_tunes_json below.